proto/flow.proto

```diff
@@ -32,7 +32,8 @@ message Record {
 
   uint64 bytes = 8;
   uint64 packets = 9;
-  string interface = 10;
+  uint32 flags = 10;
+  string interface = 11;
```

Comment:
Changing the interface index would create breaking changes in flowlogs-pipeline.

This should be:
```
string interface = 10;
uint32 flags = 11;
```

Reply:
This is something critical to avoid that any update to a newer agent (without updating FLP) breaks the whole netobserv product.